### DISCUSSION TITLE
Missing folder icons on fresh install in dolphin.

### DISCUSSION THREAD

**Discussion by KAGEYAM4** [prasanthrangan/hyprdots#206] [Q&A] [answered]
After using hypdots, i was missing icons in dolphin , so at that time i just installed plasma-desktop and icons were showing. But i only use hyprland and plasma was a bloat , so i uninstalled it again. Now icons are gone again. 
![image](https://github.com/prasanthrangan/hyprdots/assets/75798544/378160c5-4fd6-4925-88da-ca8c52a2d6fd)

Which Package is responsible for icons ?

**Comment by T-Crypt** [comment]
Icons come from Source/arcs 

You should just run the install script again to un-pack the tar.gz files that contain the icons for each theme and then it places them in /.icons 

Icon_(color).tar.gz

**Reply by KAGEYAM4** [reply]
![image](https://github.com/prasanthrangan/hyprdots/assets/75798544/9ec7ee01-7f31-4a12-a35f-d5d4317d6844)
Icons are there, i think it has something to with package.

**Reply by prasanthrangan** [reply, marked as answer]
Hi, are the QT variables set in hyprland.conf? try setting the icons with qt5ct as well and see if it works!

**Reply by KAGEYAM4** [reply]
it worked. Thank you.

**Reply by Edesem** [reply]
Can you please explain @prasanthrangan what it means to set the qt vars? 
![image](https://github.com/prasanthrangan/hyprdots/assets/93417266/4a176e61-8a2b-4b64-9ce0-0e5e3de9daf9)
Is this what you mean? If so my icons are still missing

**Reply by KAGEYAM4** [reply]
@Edesem  quoting from above ->
```
Icons come from Source/arcs

You should just run the install script again to un-pack the tar.gz files that contain the icons for each theme and then it places them in /.icons

Icon_(color).tar.gz
```

**Reply by kRHYME7** [reply]
Or you can run ```  ./restore_fnt.sh ```

**Reply by Edesem** [reply]
Did it, rebooted, no cigar @kRHYME7

**Reply by Edesem** [reply]
> @Edesem quoting from above ->
> 
> ```
> Icons come from Source/arcs
> 
> You should just run the install script again to un-pack the tar.gz files that contain the icons for each theme and then it places them in /.icons
> 
> Icon_(color).tar.gz
> ```

Did this as well, still missing

**Reply by kRHYME7** [reply]
@Edesem  What theme has missing dolphin icons?

**Reply by Edesem** [reply]
@kRHYME7  All of them, changing themes doesn't change anything, including whether it is dark mode or not, it almost looks unthemed

**Reply by kRHYME7** [reply]
ok please follow the latest commits first. 

It has different structure so I can't really help you with the outdated one

**Reply by Edesem** [reply]
Updated, but turns out everything is broken at the moment as I am sure you are aware. Icons work though :) but no waybar and drun is unthemed lmao

**Reply by kRHYME7** [reply]
hmm? how did you update BTW ?

**Reply by Edesem** [reply]
The way that it was detailed to update
```
cd ~/Hyprdots/Scripts
git pull
./install.sh -r
```

**Comment by prasanthrangan** [comment]
Hey @Edesem ,

Please try reinstalling as below, as you might be missing one of these `qt5ct`, `qt6ct`, `kvantum`, `kvantum-qt5` packages
```
cd ~/Hyprdots/Scripts
git pull
./install.sh
```

also please check these,

1. clear these directories if it exists, as they are deprecated -
```
ll ~/.config/hyprdots
ll ~/.config/swww
ll ~/.cache/hyprdots
```

2. check if the new directories exists -
```
ll ~/.config/hyde
ll ~/.cache/hyde
ll ~/.local/share/bin
```

3. verify if `QT_QPA_PLATFORMTHEME` is set in `~/.config/hypr/hyprland.conf` as below
https://github.com/prasanthrangan/hyprdots/blob/6367054846d0107035414a741b5e10712abb1d5c/Configs/.config/hypr/hyprland.conf#L50

4. finally, try running `testrunner.sh --verbose` and share the output.

**Reply by Edesem** [reply]
All packages you listed are installed and up-to-date.

I updated my Hyprdots,  got rid of all deprecated directories, confirmed all new directories exist, verified the platform theme to correct.

![image](https://github.com/prasanthrangan/hyprdots/assets/93417266/f6d9301c-7973-4da3-a108-2670e4f42237)
![image](https://github.com/prasanthrangan/hyprdots/assets/93417266/51f9863f-3d39-48bc-9140-08dfe8bed208)

This is when I press super + L-Shift + W for wallpapers.
![image](https://github.com/prasanthrangan/hyprdots/assets/93417266/2916a5fb-c76a-4dad-b76a-4d14a3c2e783)

I can't change wallpapers at all even with Super + L-Shift + Left/Right arrow keys.

I get waybar working after doing ./install.sh but disappears after either a reset or I try to change the bar position.


Here is my output from `testrunner.sh`


`[edes@ArchTop ~]$ testrunner.sh --verbose


current theme :: "Catppuccin Mocha" :: "/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/1 rain_world.png"
wallbash status :: 0 :: disabled


Theme 1 :: ${thmList[0]}="Catppuccin Mocha" :: ${thmWall[0]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/1 rain_world.png"

// Hash Map //
:: ${wallHash[0]}="a96dac249950cef9e8d9b3331b104b5cbebf17d8" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/1 rain_world.png"
:: ${wallHash[1]}="53a79e21e526c7f809747304679db1e23fd35743" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/2 rain_world.png"
:: ${wallHash[2]}="8cb5616b5ab98323751abe8ecf44bf590c89ab75" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/3 rain_world.png"
:: ${wallHash[3]}="2ff195c46d496d34dcb048ca95cbe498b08ba927" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/aesthetic_deer.png"
:: ${wallHash[4]}="ab56ddd93aeb7cf68b840dd1b57a4cde4282c084" :: ${wallList[4]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/apex_octane.jpg"
:: ${wallHash[5]}="7424188fa60cb0987eeb06f9ff7774dfda1a66a5" :: ${wallList[5]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/black_hole_by_kurzgesagt.png"
:: ${wallHash[6]}="265966daf9cc45aba71c70740c93c1d6e7ee0875" :: ${wallList[6]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/boat_abondoned.jpg"
:: ${wallHash[7]}="ac824251f158c22a3f480bdcaa41632511761278" :: ${wallList[7]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/cat_leaves.png"
:: ${wallHash[8]}="c348f05d3014e659b4f1bf3e8fe2ca087cb5ae4b" :: ${wallList[8]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/escape_velocity.jpg"
:: ${wallHash[9]}="d5e4e767c35b91696bff1ff90523be469d8baf02" :: ${wallList[9]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/evening_sky.png"
:: ${wallHash[10]}="0490d628c599bbd962b2564ed2938271425fe61e" :: ${wallList[10]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/firewatch.jpg"
:: ${wallHash[11]}="e2383d8d13626545b2853baad6439078d57f86c3" :: ${wallList[11]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/forest_dark_winter.jpg"
:: ${wallHash[12]}="de3d3e07eb38d08d891499cfdcc8ea439cd99311" :: ${wallList[12]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/hanged_man_tree.jpg"
:: ${wallHash[13]}="aa207e86eb392a98024036547a08a70f622802c2" :: ${wallList[13]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/rebellious_tower.png"
:: ${wallHash[14]}="11d8015d89fe194ba6bf681127db88d5bd36056b" :: ${wallList[14]}="/home/edes/.config/hyde/themes/Catppuccin Mocha/wallpapers/switch_swirl.jpg"



Theme 2 :: ${thmList[1]}="Catppuccin Latte" :: ${thmWall[1]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/abstract_blured.jpg"

// Hash Map //
:: ${wallHash[0]}="dcf815f42b3a41d7f1cc7c5750f0eb5358451125" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/abstract_blured.jpg"
:: ${wallHash[1]}="f0c55ed7b34b181ff97555177346b27927add062" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/cat_minimal.jpg"
:: ${wallHash[2]}="4bca9da4c1cb6767a3509b8331721618c7f86f77" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/chainsaw_angel_lake.jpg"
:: ${wallHash[3]}="330ff154a8d484b4c17ea8e07c357707c8ebd752" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/delorean_pastel.png"
:: ${wallHash[4]}="fbf6e747fc40438ab5472c1e017f97dba558d26d" :: ${wallList[4]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/flying_out.png"
:: ${wallHash[5]}="7972876d4c79a477889f1372f5c253f593e015da" :: ${wallList[5]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/gameboy.png"
:: ${wallHash[6]}="fe337b6a857e79803d30c584c8da9a32777fb1d2" :: ${wallList[6]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/jormungandr.jpg"
:: ${wallHash[7]}="7278f95a110184c1c90f0f67a50332132b50036e" :: ${wallList[7]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/reindeer_forest.jpg"
:: ${wallHash[8]}="fc568d71c6f7db6292c378ada4ee485295b1af67" :: ${wallList[8]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/rocket_launch.png"
:: ${wallHash[9]}="c46db0f336507de4264bc398894bf9a92b76e1c9" :: ${wallList[9]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/samurai_bebop.png"
:: ${wallHash[10]}="1f38381748ae86d76e384635be811fe10ca15fbe" :: ${wallList[10]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/samurai_strike.jpg"
:: ${wallHash[11]}="7f6fb59f8adcc9b312c5fad5c884d394cc656b3c" :: ${wallList[11]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/sekiro.jpg"
:: ${wallHash[12]}="aae00db20dffc074a9f363827571472e046ede6c" :: ${wallList[12]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/shaded_landscape.jpg"
:: ${wallHash[13]}="c03f8740f967c1efd9e99d1ad540fb845f6056fd" :: ${wallList[13]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/waifu_angel.jpg"
:: ${wallHash[14]}="83a2c187649dd89c8363addf91c8cfe41cc47d10" :: ${wallList[14]}="/home/edes/.config/hyde/themes/Catppuccin Latte/wallpapers/waifu_jasmin.jpg"



Theme 3 :: ${thmList[2]}="Rosé Pine" :: ${thmWall[2]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/burning_cherry.jpeg"

// Hash Map //
:: ${wallHash[0]}="6e7c83d2535f7a8900f8534f4c8973fdb077afb4" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/burning_cherry.jpeg"
:: ${wallHash[1]}="bba5b434cec3f0be22c42171f7d922841952eb41" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/chainsaw_makima.png"
:: ${wallHash[2]}="9afddbbce52054551250201f8ae7b3d520f97b3f" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/dark_purple_kimono.png"
:: ${wallHash[3]}="aca24eaaf4b987a9c5b80dbcf5d8c22e8ec1adcf" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/ferris_wheel.jpg"
:: ${wallHash[4]}="485fc6158e2b7054f8fa50b9ed5c1f884f43c474" :: ${wallList[4]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/japan_mountain_pink.jpg"
:: ${wallHash[5]}="892e3dabb25b64d412316da80c6684d77e8b2cce" :: ${wallList[5]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/japan_tower_pink.jpg"
:: ${wallHash[6]}="52fad36e87b66e29373f66e20e05a30e2dbf452b" :: ${wallList[6]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/lakeside_sunset.png"
:: ${wallHash[7]}="3e01f79806e38f9ec4c74a1492e76bfe494bbdca" :: ${wallList[7]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/mountain_dragon_pink.jpg"
:: ${wallHash[8]}="98209ef02201224687eae20616869bbd2993847b" :: ${wallList[8]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/mountain_sunset.jpg"
:: ${wallHash[9]}="c822ee496282ff28af9f1bdb4763f94d38bdebf3" :: ${wallList[9]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/mushrooms_1.png"
:: ${wallHash[10]}="2f86d677da5a97084985050aa5f4c38b94c46ac2" :: ${wallList[10]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/mushrooms_2.png"
:: ${wallHash[11]}="2f25e0169fb85a0824e8f2605e55c67c0425f607" :: ${wallList[11]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/mushrooms_3.png"
:: ${wallHash[12]}="414cca53b662b8fe4bb05b4e0c47d93c7c5d93de" :: ${wallList[12]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/mushrooms_4.png"
:: ${wallHash[13]}="d9c8b5d8a8fe329812c8ca0673e24e144756fc37" :: ${wallList[13]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/nimbus.gif"
:: ${wallHash[14]}="254d1988cc2cf1743b9505d7d811a4fcca040f94" :: ${wallList[14]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/red_purple.jpg"
:: ${wallHash[15]}="d843921bdc36e7cedd0dc8e9b311cf7ad8350b93" :: ${wallList[15]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/waifu_pink.png"
:: ${wallHash[16]}="4a2a7da6bf5d696ea406c7985efeb3ceb52eacf9" :: ${wallList[16]}="/home/edes/.config/hyde/themes/Rosé Pine/wallpapers/warm_afternoon.jpeg"



Theme 4 :: ${thmList[3]}="Tokyo Night" :: ${thmWall[3]}="/home/edes/.config/hyde/themes/Tokyo Night/wallpapers/cat_lofi_cafe.jpg"

// Hash Map //
:: ${wallHash[0]}="1c9b6ef5cd0580932ede29035341eba3d8ba9126" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Tokyo Night/wallpapers/cat_lofi_cafe.jpg"
:: ${wallHash[1]}="3011bf47a85cbc7d092992df2b03d3d90487ebb5" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Tokyo Night/wallpapers/edger_lucy_neon.jpg"
:: ${wallHash[2]}="6d75ae5fc4beab599466eb7d379c50b8036fc680" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Tokyo Night/wallpapers/gas_station.gif"
:: ${wallHash[3]}="aff85b1d85a67c8b637e7e3e072864e8551c1169" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Tokyo Night/wallpapers/lowpoly_street.png"
:: ${wallHash[4]}="ad21d173db5e83ae91e70d2c9274a2ffab603151" :: ${wallList[4]}="/home/edes/.config/hyde/themes/Tokyo Night/wallpapers/street.gif"
:: ${wallHash[5]}="7cd12a23084b0ad1e02b9053f9322afbd35cfb10" :: ${wallList[5]}="/home/edes/.config/hyde/themes/Tokyo Night/wallpapers/tokyo_pink.png"



Theme 5 :: ${thmList[4]}="Material Sakura" :: ${thmWall[4]}="/home/edes/.config/hyde/themes/Material Sakura/wallpapers/material_pastel_1.jpg"

// Hash Map //
:: ${wallHash[0]}="35688683e219b75c56aa47ea9576c7e036909989" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Material Sakura/wallpapers/material_pastel_1.jpg"
:: ${wallHash[1]}="5fa29fcb3efbb806f2c2da73f676c49b4c83dda4" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Material Sakura/wallpapers/material_pastel_2.jpg"
:: ${wallHash[2]}="9223f76ebb3b8d7084cddf83447b2346e56ceb79" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Material Sakura/wallpapers/material_pastel_3.jpg"
:: ${wallHash[3]}="4f82ad586fae85f1b73ada33862aa62446a50b99" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Material Sakura/wallpapers/material_pastel_4.jpg"
:: ${wallHash[4]}="46f8a62186ec655a2dc5ed198549a4715812efff" :: ${wallList[4]}="/home/edes/.config/hyde/themes/Material Sakura/wallpapers/material_pastel_5.jpg"
:: ${wallHash[5]}="0755cfc48c451dcaff7eb8e77e27747f45c228b5" :: ${wallList[5]}="/home/edes/.config/hyde/themes/Material Sakura/wallpapers/material_pastel_6.jpg"
:: ${wallHash[6]}="654890440fe529c95cc19cad6ade40de839d2b1e" :: ${wallList[6]}="/home/edes/.config/hyde/themes/Material Sakura/wallpapers/material_pastel_7.jpg"
:: ${wallHash[7]}="e4054c124b8476945da9c392b61cd8b0430c236e" :: ${wallList[7]}="/home/edes/.config/hyde/themes/Material Sakura/wallpapers/material_pastel_8.jpg"



Theme 6 :: ${thmList[5]}="Graphite Mono" :: ${thmWall[5]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/chainsawman_sketch.png"

// Hash Map //
:: ${wallHash[0]}="defdcaf4fd56a46adfa5b83bef2d89631ef2eed0" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/chainsawman_sketch.png"
:: ${wallHash[1]}="e1d3f04e637cf45ff2f8181d4adac6ea7dd47035" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/gojo.png"
:: ${wallHash[2]}="8eeff6b8eac86bc20ac77a7306c75a5f2c9dbbc2" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/japan_lake.jpg"
:: ${wallHash[3]}="c6a39e6ce4186938f1411c0d01892df318e8f10b" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/limbo.jpg"
:: ${wallHash[4]}="9115a3158b97c1ee82679b446dc06b0de25baf5a" :: ${wallList[4]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/min_mountain.jpg"
:: ${wallHash[5]}="8c47111caa844b309a3ad4da0bbd482fffe8869c" :: ${wallList[5]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/mountain_snow.jpg"
:: ${wallHash[6]}="de833d469d3d075d80b4336dab74c9e9f8384991" :: ${wallList[6]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/nier_2b.jpg"
:: ${wallHash[7]}="69b869f09df0f197c78f270bb481b67b41c82855" :: ${wallList[7]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/nier_mech.jpg"
:: ${wallHash[8]}="f9abed471c0e5970cc0044f08314161439dd87ce" :: ${wallList[8]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/rocket.jpg"
:: ${wallHash[9]}="d9308408d4b272c95923ad28f5f347337eb71ef3" :: ${wallList[9]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/spring_bloom.jpg"
:: ${wallHash[10]}="1d41a0ebd9c0462c2c96f0ae18bcded9ebd2cc4e" :: ${wallList[10]}="/home/edes/.config/hyde/themes/Graphite Mono/wallpapers/wall_paint.jpg"



Theme 7 :: ${thmList[6]}="Decay Green" :: ${thmWall[6]}="/home/edes/.config/hyde/themes/Decay Green/wallpapers/arcade_decay_red.png"

// Hash Map //
:: ${wallHash[0]}="6364f341e65de0b3d4619fb9b2d8c477742c4328" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Decay Green/wallpapers/arcade_decay_red.png"
:: ${wallHash[1]}="883655c211446a72e2eb01651c6f87aea8123ecf" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Decay Green/wallpapers/aurora_borealis.png"
:: ${wallHash[2]}="4fedbbb975e9a5189aea8a1c5feb0168b6382c04" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Decay Green/wallpapers/blue_swirl.png"
:: ${wallHash[3]}="45d5417535d37829d44b71a1b357b4bfac48d789" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Decay Green/wallpapers/moments_before_desk.png"
:: ${wallHash[4]}="fcee5fa6bc47e39674be2c0d9639496fe1208b3e" :: ${wallList[4]}="/home/edes/.config/hyde/themes/Decay Green/wallpapers/night_city.png"
:: ${wallHash[5]}="c4e45a33a5a498be924e5b992d5851c11ad1bfbb" :: ${wallList[5]}="/home/edes/.config/hyde/themes/Decay Green/wallpapers/relaxed_mario.png"



Theme 8 :: ${thmList[7]}="Edge Runner" :: ${thmWall[7]}="/home/edes/.config/hyde/themes/Edge Runner/wallpapers/cyberpunk_dave.png"

// Hash Map //
:: ${wallHash[0]}="145f5e9d5748b92cdc66a7fda97f1dcd913b9732" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Edge Runner/wallpapers/cyberpunk_dave.png"
:: ${wallHash[1]}="743a9648fe4f09a8e8d32321a01e37e4f95c99a4" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Edge Runner/wallpapers/david.jpg"
:: ${wallHash[2]}="5defe9945711af8db56d0fd29280144d8695b16a" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Edge Runner/wallpapers/lucy_1.jpg"
:: ${wallHash[3]}="f1843c3f0b194d6dcf63053945d9b38e7ad81466" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Edge Runner/wallpapers/lucy_2.png"
:: ${wallHash[4]}="834723916dec62e494d21866a561b2c8b08965c3" :: ${wallList[4]}="/home/edes/.config/hyde/themes/Edge Runner/wallpapers/lucy_rebecca.jpg"



Theme 9 :: ${thmList[8]}="Frosted Glass" :: ${thmWall[8]}="/home/edes/.config/hyde/themes/Frosted Glass/wallpapers/blue_desert.png"

// Hash Map //
:: ${wallHash[0]}="1c43b4729257d977ded0bae7e1a3ed76f05c6738" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Frosted Glass/wallpapers/blue_desert.png"
:: ${wallHash[1]}="774d8335281e473eb23db9497adf1dd2ed1e5fce" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Frosted Glass/wallpapers/cloudy_crescent.png"
:: ${wallHash[2]}="0a8864cc02dd2bdcf42e5a02e309e02c07c8e4cc" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Frosted Glass/wallpapers/cloudy_mountain.png"
:: ${wallHash[3]}="60b4fc342f37c558de8274227c304dba0ad43e8d" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Frosted Glass/wallpapers/lake_mountain.jpg"
:: ${wallHash[4]}="c6c6fdf701f509194eb1772786c24de3ce86d7ec" :: ${wallList[4]}="/home/edes/.config/hyde/themes/Frosted Glass/wallpapers/min_forest.jpg"
:: ${wallHash[5]}="1b342d70bba00a0f937acf30ab394a1dce93a8f9" :: ${wallList[5]}="/home/edes/.config/hyde/themes/Frosted Glass/wallpapers/teal_mountain.png"



Theme 10 :: ${thmList[9]}="Gruvbox Retro" :: ${thmWall[9]}="/home/edes/.config/hyde/themes/Gruvbox Retro/wallpapers/finalizer.png"

// Hash Map //
:: ${wallHash[0]}="e90ced8ca3384bd031fec81f1f4cf78ba735f783" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Gruvbox Retro/wallpapers/finalizer.png"
:: ${wallHash[1]}="8d8fb46d67b94bcdcea9a1c2336b73de2fc8c074" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Gruvbox Retro/wallpapers/future_town.jpg"
:: ${wallHash[2]}="e15362e6258f8e48850e9746c086ae7b36abdabb" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Gruvbox Retro/wallpapers/lady.png"
:: ${wallHash[3]}="118b1a52f5d6e1ce528d4cf1502e8b7e737646df" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Gruvbox Retro/wallpapers/minimal_squares.png"
:: ${wallHash[4]}="9e3fd92509271431db29d91435321966a66de434" :: ${wallList[4]}="/home/edes/.config/hyde/themes/Gruvbox Retro/wallpapers/misty_forest.jpg"
:: ${wallHash[5]}="d8d9cbf4b6b9976e1670ddb50819056ba938a8c0" :: ${wallList[5]}="/home/edes/.config/hyde/themes/Gruvbox Retro/wallpapers/night_moon.png"
:: ${wallHash[6]}="e2c55ca5831ad497c7b79a27131db53d96b36e6e" :: ${wallList[6]}="/home/edes/.config/hyde/themes/Gruvbox Retro/wallpapers/patterns.jpg"
:: ${wallHash[7]}="19ea2631ecf649d73ea199352d3116608009f73a" :: ${wallList[7]}="/home/edes/.config/hyde/themes/Gruvbox Retro/wallpapers/street.png"
:: ${wallHash[8]}="0257ebfb545e74ae2bfa36680b0459f78247309b" :: ${wallList[8]}="/home/edes/.config/hyde/themes/Gruvbox Retro/wallpapers/wall.jpg"



Theme 11 :: ${thmList[10]}="Synth Wave" :: ${thmWall[10]}="/home/edes/.config/hyde/themes/Synth Wave/wallpapers/beach.jpg"

// Hash Map //
:: ${wallHash[0]}="b61243b2eccbe96a88874f5b3a86e280676055b7" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Synth Wave/wallpapers/beach.jpg"
:: ${wallHash[1]}="c22eda1fa6dc288a4ff49630e97f76009c640636" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Synth Wave/wallpapers/glitch fox.png"
:: ${wallHash[2]}="8a04a32314fbfc1739bb94a6d25c81d3824c5d98" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Synth Wave/wallpapers/palm trees.jpg"
:: ${wallHash[3]}="54e0f4e84a636e01354d62f806bd1b6802fe4322" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Synth Wave/wallpapers/waves.jpg"



Theme 12 :: ${thmList[11]}="Nordic Blue" :: ${thmWall[11]}="/home/edes/.config/hyde/themes/Nordic Blue/wallpapers/forest.jpg"

// Hash Map //
:: ${wallHash[0]}="0cfe65d58ef778f0a1faa986d77340e10c9fc75b" :: ${wallList[0]}="/home/edes/.config/hyde/themes/Nordic Blue/wallpapers/forest.jpg"
:: ${wallHash[1]}="829df5651c32d996423811f307cecb9f14abf79c" :: ${wallList[1]}="/home/edes/.config/hyde/themes/Nordic Blue/wallpapers/mountains.jpg"
:: ${wallHash[2]}="dd34ba29f654b712724fdb966f568dad808c8967" :: ${wallList[2]}="/home/edes/.config/hyde/themes/Nordic Blue/wallpapers/Nordic-Heroin.png"
:: ${wallHash[3]}="584dc0edb332d16ad598f406cbd3ad75a9bd1276" :: ${wallList[3]}="/home/edes/.config/hyde/themes/Nordic Blue/wallpapers/range.png"
:: ${wallHash[4]}="86435def6ead1868a0571add7f4ce6c75798ac66" :: ${wallList[4]}="/home/edes/.config/hyde/themes/Nordic Blue/wallpapers/sunset.jpg"
:: ${wallHash[5]}="410129eb9b811a7a03a07649d0746140e1c3aa96" :: ${wallList[5]}="/home/edes/.config/hyde/themes/Nordic Blue/wallpapers/underwater.png"
:: ${wallHash[6]}="c6fe2d2a3559fcbff7058eb5314ee87ee81d8f06" :: ${wallList[6]}="/home/edes/.config/hyde/themes/Nordic Blue/wallpapers/wild.png"`

**Reply by prasanthrangan** [reply]
Hey @Edesem , these seems to be multiple issues here... but lets check one by one.

## 1. script path
Please ensure the script path in  `~/.config/hypr/hyprland.conf` is set to `$HOME/.local/share/bin` as below,
https://github.com/prasanthrangan/hyprdots/blob/b1a53c88ad4f6b34903112551bf70e9740311e28/Configs/.config/hypr/hyprland.conf#L11

## 2. windowrules
This line in `~/.config/hypr/windowrules.conf` file is causing issues, maybe you're on old hyprland version? If so, please remove this line.
https://github.com/prasanthrangan/hyprdots/blob/b1a53c88ad4f6b34903112551bf70e9740311e28/Configs/.config/hypr/windowrules.conf#L13

## 3. wallpaper
To check why the wallpaper change fails, please try running `swwwallpaper.sh -n` from terminal and share the output. For wallpaper select menu try running `swwwallselect.sh`.

## 4. waybar
To check why the waybar change fails, please try running `wbarconfgen.sh n` from terminal and share the output.

**Reply by Edesem** [reply]
1. path is correct
2. deleted line
3. Wallpaper works correctly and the select menu is correct, it seems when I try running it through the keybinds thats when it is broken.
4. It works even after restart only if I dont change the layout to which I have to rerun the command


I check my keybinds config and its all correct soooo idk. What is the easiest way I can uninstall the whole thing and reinstall because I think there is something really corrupt or messed up in my configs. Unless you know another thing to try to fix it

**Reply by kRHYME7** [reply]
@Edesem  ``` grep -i "\$scrPath" ~/.config/hypr/*.conf ```
trace why your script path is broken

![image](https://github.com/prasanthrangan/hyprdots/assets/53417443/48552f8e-b90d-4292-9d9a-69c51a603cfd)

**Reply by Edesem** [reply]
Okay seems everything is finally fixed. I change all my script paths to be $scrPath instead of ~/.config/hypr/script/"blahblahblah" and it seems everything works now! Lol that was frustrating
